src/components/Comments.js

```diff
@@ -0,0 +1,268 @@
+        if (response && response.documents) {
+          response.documents.forEach((doc) => {
+            commentList.push(Comment.fromDoc(doc));
+          });
+          this.setState(
+            {
+              commentList,
+              comment: '',
+            },
+          );
+        } else if (error) {
+          this.setState({
+            error,
+          });
+        }
+      });
+    }
+  }
+
+  deleteComment(id: string) {
+    const jsonDoc = `{ "id" : "${id}", "mode" : "DELETE" }`;
+    this.context.searcher.search.addOrDeleteDocument(JSON.parse(jsonDoc), this.getComments);
+  }
+
+  createFormattedCommentList() {
+    const { commentList } = this.state;
+    const loggedInUser = AuthUtils.getUserName(AuthUtils.getSavedUser());
+    const formattedCommentList = [];
+    if (commentList && commentList.length > 0) {
+      commentList.forEach((comment) => {
+        const isCommentByLoggedInUser = (comment.username === loggedInUser);
+        const removeCommentOption = isCommentByLoggedInUser && (
+          <Glyphicon
+            glyph="trash"
+            onClick={() => {
+              this.deleteComment(comment.id);
+            }}
+            style={{ fontSize: '1em', color: '#003c7e', cursor: 'pointer' }}
+            title="Delete this comment"
+          />
+        );
+        formattedCommentList.push(
+          <div key={comment.id}>
+            <div
+              style={{ color: '#484848', backgroundColor: '#F5F5F5', borderRadius: '5px' }}
+              title={comment.timestamp}
+            >
+              <div style={{ padding: '0.5em' }}>
+                <b> <i> {comment.username} </i> </b>
+                <div style={{ float: 'right' }}>
+                  { removeCommentOption }
+                </div>
+                <br />
+                <span style={{ whiteSpace: 'pre-wrap' }}>
+                  {comment.text}
+                </span>
+              </div>
+            </div>
+            <br />
+          </div>,
+        );
+      });
+    }
+    return formattedCommentList;
+  }
+
+  showCommentModal() {
+    this.setState({
+      showCommentModal: true,
+    });
+  }
+
+  hideCommentModal() {
+    this.setState({
+      showCommentModal: false,
+    });
+  }
+
+  captureComment(e: Event) {
+    if (e.target instanceof HTMLTextAreaElement) {
+      this.setState({
+        comment: e.target.value,
+      });
+    }
+  }
+
+  saveComment() {
+    const { docId } = this.props;
+    const { comment } = this.state;
+    const { searcher } = this.context;
+    const username = AuthUtils.getUserName(AuthUtils.getSavedUser());
+    const d = new Date();
+    const loggedDateTime = d.toISOString();
+    const id = username.concat(loggedDateTime);
+    const body = {
+      fields: {
+        comment_s: [comment],
+        docId_s: [docId],
+        username_s: [username],
+        date: [loggedDateTime],
+        table: ['comments'],
```

Comment:
Wouldn't the users need to create such a zone first? I think it makes sense to have a separate zone considering we don't want these docs to show as a part of the search results. 

If we configure such a zone by default and use it for saved searches, comments and briefcases (upcoming PR that would add the ability to store search results in baskets/buckets/briefcases), it would make things more consistent. Thoughts?